### ISSUE TITLE
Publish for Scala 2.13.0 (yes, it's now released!)

### ISSUE THREAD

**Issue by ngocdaothanh** [scalatest/scalatest#1601]
https://github.com/scala/scala/releases

**Comment by joan38** [comment]
Here is the list of trial PRs:
scalatest/scalatest#1602
scalatest/scalatestplus-jmock#3
scalatest/scalatestplus-mockito#5
scalatest/scalatestplus-selenium#3
scalatest/scalatestplus-easymock#3
scalatest/scalatestplus-testng#4
scalatest/scalatestplus-junit#5
scalatest/scalatestplus-scalacheck#9

**Comment by cheeseng** [comment]
@ngocdaothanh @joan38 Thanks for helping, I'll going to start to work on the build today.

**Comment by joan38** [comment]
@cheeseng feel free to take the branches and do what it needs to be done

**Comment by cheeseng** [comment]
@joan38 Unfortunately it seems like we're still being blocked by scalacheck build for 2.13.0.

For now, do you mind to sign the following CLA (required by Artima Inc.): 

http://www.artima.com/cla/contributorsLicenseAgreement.pdf

and email back to cla@artima.com ?  We apologize for any inconveniences caused, we have been wanting to automate it but somehow it still didn't happen.

**Comment by joan38** [comment]
The good news is that I'm the one who did the PR in Scalacheck too:
https://github.com/rickynils/scalacheck/pull/479

CLA sent.

**Comment by cheeseng** [comment]
@joan38 Received the CLA with thanks!

I'll release scalatest 3.0.x for Scala 2.13 once ScalaCheck build is available, actually I can start with 3.1.x build now first (the one that you submitted PR) since its scalacheck dependency is separated out to scalatestplus-scalacheck already.

I'll start with your PR branch, your help is highly appreciated!  Thank you!

**Comment by joan38** [comment]
No worries. Mind the fact that I commented out all the Scala native projects because it was complaining that it can't compile with any Scala >2.11

**Comment by cheeseng** [comment]
@joan38  Yes, I'll probably bring back that one, we're just using ++2.11.12 whenever we need to build for scala-native currently.

**Comment by bvenners** [comment]
@joan38 Say can you resend your CLA with the blank lines filled in? I.e. we need more than just the signature. Tnx for all of the contributions!

**Comment by joan38** [comment]
@bvenners I filled the blank lines with my PDF viewer (Preview on Mac), your PDF viewer must not show them. So I sent it back passing it throw the PDF printer.

**Comment by SethTisue** [comment]
ScalaCheck 1.14.0 has now been published for Scala 2.13

**Comment by cheeseng** [comment]
@SethTisue @joan38 @ngocdaothanh Fyi 3.1.0-SNAP13 is up, now I am working on releasing 3.0.8.

**Comment by joan38** [comment]
@SethTisue are you sure? I can see it on https://mvnrepository.com/artifact/org.scalacheck/scalacheck

**Comment by dwijnand** [comment]
mvnrepository.com lag.  See http://repo1.maven.org/maven2/org/scalacheck/scalacheck_2.13/

**Comment by SethTisue** [comment]
https://repo1.maven.org/maven2/org/scalatest/scalatest_2.13/3.1.0-SNAP13/ :+1:
https://repo1.maven.org/maven2/org/scalatest/scalatest_2.13/3.0.8/ :+1:

I think this ticket can be closed now.

**Comment by yanns** [comment]
I cannot find the git tag for `3.0.8`. Someone forgot to push it?

**Comment by cheeseng** [comment]
@yanns Oh yes I forgot, sorry, I'll do that tonight after dinner.

Sorry for inconvenience caused.

**Comment by cheeseng** [comment]
@yanns I just created it here: 

https://github.com/scalatest/scalatest/releases/tag/release-3.0.8

Cheers.

**Comment by SethTisue** [comment]
seems like this could be closed now
